usr/lib/systemd/system/greenboot.service

```diff
@@ -0,0 +1,17 @@
+#  This file is part of greenboot.
+#
+#  greenboot is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=greenboot Success Scripts Runner
+BindsTo=greenboot.target
```

Comment:
This works with `Requires` too. Or `RequiredBy` on the target. BindsTo is a stronger binding. I was thinking it might make sense because once we have Watchdog check services and they make the target fail, this should be propagated to greenboot.service, too. BindsTo does that.

Reply:
ok, we'll use `Requires`, as discussed on IRC